src/content/self-hosted/install/releases.mdx

```diff
@@ -26,6 +37,11 @@ This version is compatible with Kubernetes versions 1.24 to 1.27.
 
 - Fix private registry credentials integration for registries with custom ports (other that the default 443)
 
+### Features
+
+- The Okteto CLI used for Pipeline installation has been upgraded to version [2.22.3](https://github.com/okteto/okteto/releases/tag/2.22.3)
```

Comment:
@maroshii This was added in the wrong version, right? CLI `2.22.x` is included as part of `1.15.0` and not as part of `1.14.2`, right?